tool/verify_pub_score.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+set -e  # Exit on any command failure
```

Comment:
Bit of cleanup to this script as I noticed it didn't exit 1 if any errors occurred during execution.